index.bs

```diff
@@ -21,15 +21,15 @@ It can nevertheless be used as the main source for developers, as it links to th
 
 ## Our goal ## {#our-goal}
 
-The [Partnership for Carbon Transparency (PACT)](https://carbon-transparency.com) seeks to accelerate decarbonization through the creation of transparency on emissions in the value chain.
+The [Partnership for Carbon Transparency (PACT)](https://carbon-transparency.com) seeks to accelerate decarbonisation through the creation of transparency on emissions in the value chain.
```

Comment:
I believe that WBCSD uses American spelling (at least for the time being), so here it should be 'decarbonization', no?

Reply:
Fixed!